tooling/maven/camel-package-maven-plugin/src/main/resources/component-options.mvel

```diff
@@ -8,6 +8,6 @@ The @{title} component supports @{componentOptions.size()} options, which are li
 [width="100%",cols="2,5,^1,2",options="header"]
 |===
 | Name | Description | Default | Type
-@foreach{row : componentOptions}| *@{row.getShortName(25)}* (@{row.shortGroup}) | @{util.escape(row.description)} | @{row.getShortDefaultValue(20)} | @{row.getShortJavaType(25)}
+@foreach{row : componentOptions}| *@{row.getShortName(25)}* (@{row.shortGroup}) @{row.description.?contains("\n") ? "a" : ""}| @{util.escape(row.description)} | @{row.getShortDefaultValue(20)} | @{row.getShortJavaType(25)}
```

Comment:
What does this "a" do?

Reply:
@davsclaus have a look at the [Asciidoctor manual](https://asciidoctor.org/docs/user-manual/#cell) this helps in rendering the Asciidoc column content, without it it would render verbatim.

Reply:
Okay is there a way to document this better in the source code so anyone knows this